vignettes/grattan.Rmd

```diff
@@ -810,8 +812,10 @@ sample_files_all %>%
 ```{r CGT-by-age-weighted-ntile}
 set.seed(24841)
 sample_files_all %>%
+  lazy_dt %>%
```

Comment:
I was wondering whether we would need to make updates (other than to `put-data.R`) in light of the changes to `dtplyr`!